Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -711,6 +728,24 @@ bool EventsRefactorer::SearchStringInConditions(
   return false;
 }
 
+bool EventsRefactorer::SearchStringInComments(gd::ObjectsContainer& project,
+                                               gd::ObjectsContainer& layout,
+                                               gd::EventsList& events,
+                                               gd::String search,
+                                               bool matchCase) {
+  for (gd::BaseEvent event : events) {
+    for (gd::String str : event.GetAllSearchableStrings()) {
+      if (matchCase) {
+        if (str.find(search) != gd::String::npos) return true;
+      } else {
+        if (str.FindCaseInsensitive(search) != gd::String::npos) return true;
+      }
+    }

```

Comment:
This is good :) You're looking into every string. As soon as one contains the search, we return true!